modules/platform/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java

```diff
@@ -133,4 +136,27 @@ public void testThumbnailsForTIFF() throws Exception {
         assertNotNull("the TIFF thumbnail is not generated", thumbnail);
     }
 
+    @Test
+    public void testNotRecomputeThumbnailsWhenDisabled() throws IOException, OperationException {
+        Blob blob = Blobs.createBlob(FileUtils.getResourceFileFromContext("test-data/big_nuxeo_logo.jpg"), "image/jpeg",
+                StandardCharsets.UTF_8.name(), "big_nuxeo_logo.jpg");
+
+        DocumentModel fileWithThumbnail = session.createDocumentModel("/", "fileWithThumbnail", "File");
```

Comment:
The thumbnail generation is already tested before.

Reply:
yes i agree, as explained above i just keep the same thing (check the same document):
`disabling` -> no thumbnail, no conversion
`re-enabling`-> thumbnail, conversion

Reply:
Done for this case :)